src/components/Settings/SettingsTab.tsx

```diff
@@ -174,6 +182,47 @@ export const SettingsTab = ({
           disabled={false}
         />
       )}
+      {owner && owner === user?.uid && (
+        <React.Fragment>
+          <SettingRow
+            icon={'user delete'}
+            name={`Kick User`}
+            description="Kick a user out of the room. They can join again unless the room is protected with a password unknown to them."
+            disabled={true}
+            hideToggle={true}
+          />
+          <Dropdown
+            placeholder="Select the user to be kicked out."
+            fluid
+            search
+            selection
+            clearable
+            value={userToBeKicked}
+            options={participants?.map((u) => ({
+              text:
+                nameMap[u.id] +
+                u.id.slice(u.id.lastIndexOf('#')).substring(0, 4),
+              value: u.id,
+            }))}
+            onChange={(e_, data) => setUserToBeKicked(String(data.value))}
+          />
+          {userToBeKicked && (
+            <div
+              style={{
+                marginTop: 10,
+                backgroundColor: 'rgb(134 39 20)',
+                padding: 5,
+              }}
+            >
+              User{' '}
+              <span style={{ fontWeight: 'bold' }}>
+                {nameMap[userToBeKicked]}
+              </span>{' '}
+              will be kicked out after the admin settings have been saved.
```

Comment:
I think this flow of "kick the user on save" is going to be unnatural and confusing since that's not how most kicking works. I think it would be better if we surfaced it as a menu option on the members list.

Reply:
i.e. each user would have a "three dots" menu and one of the options would be "kick user" if the viewer is the room admin